src/main/java/com/firebolt/jdbc/type/JavaTypeToFireboltSQLString.java

```diff
@@ -32,7 +32,7 @@ public enum JavaTypeToFireboltSQLString {
 	ARRAY(Array.class, SqlArrayUtil::arrayToString);
 
 	private static final Pair<String[], String[]> characterToEscapedCharacterPair = new ImmutablePair<>(
-			new String[] { "\0", "\\", "'", "`" }, new String[] { "\\0", "\\\\", "\\'", "\\`" });
+			new String[] { "\0", "\\", "'" }, new String[] { "\\0", "\\\\", "\\'" });
```

Comment:
Why's this no longer escaped?

Reply:
I just give it a try and I couldnt use ` as a quote (it works as a standard character so I don't think we should escape it)

Reply:
Interesting, looks like in CH ` is equivalent to ", but we're not allowing it at all. Probably for the best.